CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ option(WITH_XDG_DIRS_FALLBACK "Use our XDG_CONFIG_DIRS fallback" ON)
 
 set(LXQT_BUILD_TOOLS_MAJOR_VERSION 0)
 set(LXQT_BUILD_TOOLS_MINOR_VERSION 3)
-set(LXQT_BUILD_TOOLS_PATCH_VERSION 1)
+set(LXQT_BUILD_TOOLS_PATCH_VERSION 2)
 set(LXQT_BUILD_TOOLS_VERSION ${LXQT_BUILD_TOOLS_MAJOR_VERSION}.${LXQT_BUILD_TOOLS_MINOR_VERSION}${LXQT_BUILD_TOOLS_PATCH_VERSION})
 
```

Comment:
Just seen: There is a dot missing between `${LXQT_BUILD_TOOLS_MINOR_VERSION}` and `${LXQT_BUILD_TOOLS_PATCH_VERSION}`…